src/components/Process/Create/index.tsx

```diff
@@ -431,7 +430,7 @@ const formToElectionMapper = (form: Process, census: Census): UnpublishedElectio
     voteType: {
       maxVoteOverwrites: 10,
     },
-    temporarySecretIdentity: form.censusType === CensusTypes.Spreadsheet && Boolean(form.voterPrivacy === 'anonymous'),
+    temporarySecretIdentity: form.censusType === CensusTypes.Spreadsheet && Boolean(form.voterPrivacy !== 'public'),
```

Comment:
This change is not OK. It will work but it could cause the secret identity feature to be enabled if the value is an incorrect value like `whatever`. Please revert the change back (note that by leaving it the way it was, it's not going to bring problems related to the privacy setting being removed)